reactor-core/src/main/java/reactor/util/context/Context0.java

```diff
@@ -24,6 +24,9 @@
 
 	static final Context0 INSTANCE = new Context0();
 
+	private Context0() {
```

Comment:
we might want to introduce new special `Context0` constants in the future (eg. to differentiate a DEFAULT vs a valued context that was emptied)